tests/std/tests/Dev11_0135139_vector_bool_comparisons/test.cpp

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <assert.h>
+#include <memory>
+#include <random>
+#include <vector>
+
+using namespace std;
+
+vector<bool> vb_from_str(const char* str) {
+    vector<bool> vb;
+
+    for (; *str != '\0'; ++str) {
+        assert(*str == '0' || *str == '1');
+
+        vb.push_back(*str == '1');
+    }
+
+    return vb;
+}
+
+enum class Ordering : int { Less = -1, Equal = 0, Greater = 1 };
+
+constexpr Ordering Lt = Ordering::Less;
+constexpr Ordering Eq = Ordering::Equal;
+constexpr Ordering Gt = Ordering::Greater;
+
+void test_comparison(const char* const left_str, const char* const right_str, const Ordering order) {
+    const auto left  = vb_from_str(left_str);
+    const auto right = vb_from_str(right_str);
+
+#ifdef __cpp_lib_concepts
+    assert((left <=> right) == (static_cast<int>(order) <=> 0));

```

Comment:
Only two `<=>` in an expression? Pffft. https://twitter.com/BarryRevzin/status/1103131654355324928 (No change requested.)